lib/google_drive/session.rb

```diff
@@ -253,7 +253,7 @@ def files(params = {}, &block)
       params = convert_params(params)
       execute_paged!(
         method: drive_service.method(:list_files),
-        parameters: { fields: '*', supports_team_drives: true }.merge(params),
+        parameters: { fields: '*', supports_team_drives: true, include_team_drive_items: true }.merge(params),
```

Comment:
It looks includeItemsFromAllDrives is preferred to includeTeamDriveItems per the Google document. Can you use includeItemsFromAllDrives instead?

Reply:
Fix parameters to `includeItemsFromAllDrives` and `supportsAllDrives`.

Reason is that `supports_team_drives` attributes used by some codes and it looks like `supportsTeamDrives` and `includeTeamDriveItems` is pairs.